src/rulesets/Base/array.jl

```diff
@@ -4,6 +4,10 @@
 
 ChainRules.@non_differentiable (::Type{T} where {T<:Array})(::UndefInitializer, args...)
 
+function frule((_, xdot), ::Type{T}, x::AbstractArray) where {T<:Array}
```

Comment:
Can we run a find + replace in files, and fix names like `xdot` into unicode `ẋ`
for consistency with the rest of the project